HTTPlayground/settings.py

```diff
@@ -1,13 +1,25 @@
 from pathlib import Path
 from sys import argv
+from jinja2 import Environment, FileSystemLoader
 
-PORT = 8000
-URL = 'localhost'
+SERVERS_INFO = {
+    'URL0': '127.0.0.1', 'PORT0': 8000,
+    'URL1': '0.0.0.0', 'PORT1': 8080  # 127.255.255.254
+}
 
-if len(argv) > 2:
-    PORT = int(argv[2])
-    URL = argv[1]
+IS_MAIN_SERVER = not int(argv[1])
+
+PORT = SERVERS_INFO.get(f'PORT{argv[1]}', SERVERS_INFO['PORT0'])
```

Comment:
I wouldn't fall back to any default value here. Usually one wants to be specific on what they want to run, so they do need to supply an explicit value.

Reply:
Ok, will remove default value.

> I wouldn't fall back to any default value here. Usually one wants to be specific on what they want to run, so they do need to supply an explicit value.